lib/validate/validate_function.js

```diff
@@ -6,9 +6,11 @@ var validate = require('./validate');
 var validateObject = require('./validate_object');
 var validateArray = require('./validate_array');
 var validateNumber = require('./validate_number');
+var unbundle = require('../util/unbundle_jsonlint');
 
 module.exports = function validateFunction(options) {
     var functionValueSpec = options.valueSpec;
+    var functionValue = options.value;
```

Comment:
why do you define this var but continue using `options.value...` throughout the function? maybe this has something to do w the what needs to be `unbundled`?